app/src/main/java/com/keylesspalace/tusky/fragment/NotificationsFragment.java

```diff
@@ -491,6 +498,84 @@ public void onLoadMore(int position) {
         }
     }
 
+    /**
+     * Called when the status {@link android.widget.ToggleButton} responsible for collapsing long
+     * status content is interacted with.
+     *
+     * @param isCollapsed Whether the status content is shown in a collapsed state or fully.
+     * @param position    The position of the status in the list.
+     */
+    @Override
+    public void onContentCollapsedChange(boolean isCollapsed, int position) {
+        if(position < 0 || position >= notifications.size()) {
+            Log.e(TAG, String.format("Tried to access out of bounds status position: %d of %d", position, notifications.size() - 1));
+            return;
+        }
+
+        NotificationViewData notification = notifications.getPairedItem(position);
+        if(!(notification instanceof NotificationViewData.Concrete)) {
+            if(notification == null) {
+                Log.e(TAG, String.format(
```

Comment:
Why would it be `null`?

Reply:
I had that because nowhere were get or set operations marked as `@NonNull` for the provided elements so I picked the "better safe than sorry" route around it.
I can remove it but PairedList (and the used List implementers in it) doesn't guarantee null safety.